browser/js/common/filters/CategoryFilter.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
```

Comment:
We may run into a issue with this if we are trying to allow users to click on individual tags on the item page to pull up items by that category